python/utils/bootstrap_utilities.py

```diff
@@ -128,7 +139,7 @@ def start_engine(self):
             manager.restore_settings(self._manager_settings)
             manager.pre_engine_start_callback = self._pre_engine_start_callback
             manager.progress_callback = self._progress_callback
-
+            raise RuntimeError("Oh SNAP!")
```

Comment:
is this left over from testing? This doesn't seem right, as the following lines will never be reached.

Reply:
Ha, I see you removed it just as I posted my review 😆